plugins/inputs/windows_event_log/windows_event_log.go

```diff
@@ -25,6 +27,8 @@ const (
 	forcePullInterval = 250 * time.Millisecond
 )
 
+var startOnlyOnce sync.Once
```

Comment:
nit: I think we might be able to move this into the `Plugin` struct since the plugin should only be initialized in the `init` function and should be the same instance wherever it's used.